pyroomacoustics/utilities.py

```diff
@@ -738,6 +738,16 @@ def goertzel(x, k):
     return y
 
 
+class Options(object):
```

Comment:
I was just wondering if something equivalent exists in [collections](https://docs.python.org/3.8/library/collections.html). Also, maybe we could add a docstring to make the purpose of this class more obvious.

Reply:
I couldn't see anything in collections that would have equivalent functionality, do you know of something? I've added some docstring in any case.

Reply:
Right, maybe we can achieve the same thing by using an [enum](https://docs.python.org/3/library/enum.html). Then, we don't provide a string as argument, but something like "DirectivityPattern.CARDIOID".

Reply:
Enum looks good 👍 I've switched to this.